### PR TITLE
BrenKt-SK1128-WEB-Email-locked_for_spam

### PR DESCRIPTION
**Pull request by BrenKt**

### CODE REVIEW

File: pages/web/emails/locked_for_spam/guide.fr-fr.md [hunk 1]
```diff
@@ -4,54 +4,92 @@ slug: bloque-pour-spam
...
 
-- Disposer d'une [offre e-mail OVHcloud](https://www.ovhcloud.com/fr/emails/){.external}.
+- Disposer d'une [offre e-mail OVHcloud](https://www.ovhcloud.com/fr/emails/).
 - Être connecté à [l'espace client OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.com/fr/&ovhSubsidiary=fr), partie `Web Cloud`{.action}.
 
-## En pratique
+## En pratique <a name="instructions"></a>
 
-### Étape 1 : vérifier le statut de l'adresse e-mail et accéder au ticket d'assistance associé
+Avant de poursuivre, si le blocage concerne une adresse e-mail de type MXplan, il est nécessaire d'identifier la version que vous possédez pour suivre le bon processus de déblocage. Vérifiez à l'aide du tableau ci-dessous comment distinguer les deux versions.
```
Comment (Kazarus4) [inline pages/web/emails/locked_for_spam/guide.fr-fr.md]:
```suggestion
Avant de poursuivre et si le blocage concerne une adresse e-mail de type MXplan, Identifiez la version que vous possédez pour suivre le bon processus de déblocage. Vérifiez à l'aide du tableau ci-dessous comment distinguer les deux versions.
```

File: pages/web/emails/locked_for_spam/guide.fr-fr.md [hunk 2]
```diff
@@ -4,54 +4,92 @@ slug: bloque-pour-spam
...
-updated: 2020-04-20
+updated: 2023-06-06
 ---
 
 **Dernière mise à jour le 16/03/2020**
 
 ## Objectif
 
-Si vous avez reçu un e-mail indiquant que l'une de vos adresses e-mail est bloquée pour spam, voici les vérifications et actions à mener.
+Lorsque votre adresse e-mail est bloquée pour SPAM, cela signifie qu'une activité suspecte a été détectée au niveau de l'envoi des e-mails depuis cette adresse. Dans cette situation vous ne pouvez plus envoyer d'e-mail depuis cette adresse e-mail, il est donc nécessaire de comprendre pourquoi une activité suspecte a été détectée et comment l'éviter.
```
Comment (Kazarus4) [inline pages/web/emails/locked_for_spam/guide.fr-fr.md]:
```suggestion
Lorsque votre adresse e-mail est bloquée pour SPAM, cela signifie qu'une activité suspecte a été détectée au niveau de l'envoi des e-mails depuis cette adresse. Dans cette situation vous ne pouvez plus envoyer d'e-mail depuis cette adresse e-mail. Vous devrez comprendre pourquoi une activité suspecte a été détectée et réaliser des actions afin d'éviter que cela se reproduise.
```

File: pages/web/emails/locked_for_spam/guide.fr-fr.md [hunk 3]
```diff
@@ -4,54 +4,92 @@ slug: bloque-pour-spam
...
+|---|---|
+|![email](images/mxplan-starter-legacy-step1.png){.thumbnail}<br> Repérez l'offre dans le cadre « Abonnement »|![email](images/mxplan-starter-new-step1.png){.thumbnail}<br>Repérez la « Référence serveur » dans le cadre « Résumé »|
 
-Connectez-vous à votre [espace client OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.com/fr/&ovhSubsidiary=fr) et dirigez-vous dans la section `Webcloud`{.action}. Cliquez sur `Microsoft`{.action} puis sur `Exchange`{.action} et sélectionnez la plateforme Exchange concernée.
+### Étape 1 : pourquoi votre adresse e-mail est bloquée pour SPAM ? <a name="step1"></a>
 
-Dirigez-vous vers l'onglet `Comptes e-mail`{.action} de votre plateforme. Si la colonne « statut » de l'adresse e-mail concernée mentionne « bloqué », cliquez sur `...`{.action} à droite du compte puis sur `Débloquer`{.action}. Passez ensuite à [l'étape 2](./#etape-2-acceder-au-ticket-dassistance_1){.external} du guide.
+Lorsqu'une activité suspecte est détectée au niveau de l'envoi des e-mails, l'adresse concernée est automatiquement bloquée. Dans cette situation vous ne pouvez plus envoyer d'e-mail depuis cette adresse e-mail.
```
Comment (Kazarus4) [inline pages/web/emails/locked_for_spam/guide.fr-fr.md]:
```suggestion
Lorsqu'une activité suspecte est détectée au niveau de l'envoi des e-mails, l'adresse concernée est automatiquement bloquée. Dans cette situation vous ne pouvez plus envoyer d'e-mails depuis cette adresse e-mail.
```

File: pages/web/emails/locked_for_spam/guide.fr-fr.md [hunk 4]
```diff
@@ -4,54 +4,92 @@ slug: bloque-pour-spam
...
 
-![spam](images/blocked-for-SPAM-01-01.png){.thumbnail}
+Tout d'abord, assurez-vous auprès du ou des utilisateurs de l'adresse e-mail bloquée qu'ils ne soient pas directement à l'origine du blocage, suite à une utilisation inhabituelle (exemple: envois massif d'e-mail). Si c'est le cas, il est nécessaire de corriger cela avant de débloquer l'adresse.
```
Comment (Kazarus4) [inline pages/web/emails/locked_for_spam/guide.fr-fr.md]:
```suggestion
Tout d'abord, assurez-vous, auprès du (des) utilisateur(s) de l'adresse e-mail bloquée, qu'il(s) n'est (ne sont) pas directement lié(s) à l'origine du blocage. Ceci suite à une utilisation inhabituelle de l'adresse e-mail (exemple: réalisation d'envois massifs d'e-mails). Si c'est le cas, vous devrez corriger la situation avant de débloquer l'adresse.
```

File: pages/web/emails/locked_for_spam/guide.fr-fr.md [hunk 5]
```diff
@@ -4,54 +4,92 @@ slug: bloque-pour-spam
...
-![spam](images/blocked-for-SPAM-01-02.png){.thumbnail}
-
-#### Pour une adresse e-mail MX plan
-
-Connectez-vous à votre [espace client OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.com/fr/&ovhSubsidiary=fr) et dirigez-vous dans la section `Webcloud`{.action}. Cliquez sur `Emails`{.action}, puis sélectionnez le nom de domaine concerné.
-
-Dirigez-vous vers l'onglet `Comptes e-mail`{.action} de votre plateforme. Si la colonne « statut » à droite de l'adresse e-mail concernée mentionne « Spam », cliquez sur cette mention puis sur `Répondre au ticket`{.action}. Passez ensuite à [l'étape 2](./#etape-2-acceder-au-ticket-dassistance_1){.external} du guide.
-
-![spam](images/blocked-for-SPAM-01-03.png){.thumbnail}
+Si l'activité suspect détecté par l'anti-spam n'a pas été initiée par le ou les utilisateurs légitimes de l'adresse e-mail, nous vous invitons à prendre les mesures ci-dessous:
```
Comment (Kazarus4) [inline pages/web/emails/locked_for_spam/guide.fr-fr.md]:
```suggestion
Si l'activité suspecte détectée par l'anti-spam n'a pas été initiée par le (les) utilisateur(s) légitime(s) de l'adresse e-mail, prennez les mesures nécessaires ci-dessous:
```

File: pages/web/emails/locked_for_spam/guide.fr-fr.md [hunk 6]
```diff
@@ -4,54 +4,92 @@ slug: bloque-pour-spam
...
 
+- Effectuer une analyse antivirus de chacun des postes utilisant l'adresse e-mail bloquée pour spam et appliquer un correctif si ces derniers sont infectés.
 
-### Étape 2 : accéder au ticket d'assistance
+- Vérifier tous les logiciels utilisant les identifiants de l'adresse e-mail bloquée pour spam (Exemple: télécopieur, logiciel métier, logiciel de messagerie).
 
-Suite à l'étape 1, vous serez alors redirigé vers la fenêtre « Mes demandes d'assistance ». Cliquez sur `...`{.action} à droite du ticket mentionnant l'objet « Account locked for spam.», puis cliquez sur `Voir le détail`{.action}. 
+### Étape 2 : vérifier le statut de l'adresse e-mail et accéder au ticket d'assistance associé
+
+Sélectionnez l'offre e-mail concernée dans les onglets suivant:
```
Comment (Kazarus4) [inline pages/web/emails/locked_for_spam/guide.fr-fr.md]:
```suggestion
Sélectionnez l'offre e-mail concernée dans les onglets suivants:
```

File: pages/web/emails/locked_for_spam/guide.fr-fr.md [hunk 7]
```diff
@@ -4,54 +4,92 @@ slug: bloque-pour-spam
...
+>>
+>> ![spam](images/blocked-for-SPAM-01-03.png){.thumbnail}
+>>
+> **MX plan - Historique**
+>>
+>> Si le blocage concerne une adresse e-mail [MXplan version historique](#instructions), il n'y a pas de ticket d'assistance. Veillez bien à consulter [l'étape 1](#step1) de ce guide avant de suivre les instructions suivante.
+>>
+>> Connectez-vous à votre [espace client OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.com/fr/&ovhSubsidiary=fr) et dirigez-vous dans la section `Webcloud`{.action}. Cliquez sur `Emails`{.action}, puis sélectionnez le nom de domaine concerné.
+>>
+>> Dirigez-vous vers l'onglet `Emails`{.action} de votre plateforme. Si la colonne « Bloqué pour SPAM » mentionne « Oui », cliquez sur cette mention puis sur `Changer le mot de passe`{.action}. Votre adresse e-mail est maintenant débloquée, vous n'avez pas besoin de suivre l'étape 3.
```
Comment (Kazarus4) [inline pages/web/emails/locked_for_spam/guide.fr-fr.md]:
```suggestion
>> Dirigez-vous vers l'onglet `Emails`{.action} de votre plateforme. Si la colonne « Bloqué pour SPAM » mentionne « Oui », cliquez sur cette mention puis sur `Changer le mot de passe`{.action}. Votre adresse e-mail est maintenant débloquée, vous n'avez pas besoin de suivre l'[étape 3](#step3).
```

File: pages/web/emails/locked_for_spam/guide.fr-fr.md [hunk 8]
```diff
@@ -4,54 +4,92 @@ slug: bloque-pour-spam
...
+>> ![spam](images/blocked-for-SPAM-01-04.png){.thumbnail}
+>>
+>> > [!warning]
+>> >
+>> > Dans de rares cas, la colonne « Bloqué pour SPAM » peut indiquer « Non » malgré le fait que l'adresse e-mail soit bloquée. Si vous avez fait le nécessaire pour sécuriser l'adresse e-mail, la solution reste la même que ci-dessus.
+
+
+### Étape 3 : accéder au ticket d'assistance <a name="step3"></a>
+
+Suite à l'étape 3, vous serez alors redirigé vers la fenêtre « Mes demandes d'assistance ». Cliquez sur `...`{.action} à droite du ticket mentionnant l'objet « Account locked for spam.», puis cliquez sur `Voir le détail`{.action}.
```
Comment (Kazarus4) [inline pages/web/emails/locked_for_spam/guide.fr-fr.md]:
```suggestion
Suite à l'étape 2, vous serez alors redirigé vers la fenêtre « Mes demandes d'assistance ». Cliquez sur le bouton `...`{.action} à droite du ticket mentionnant l'objet « Account locked for spam.», puis cliquez sur `Voir le détail`{.action}.
```

File: pages/web/emails/locked_for_spam/guide.fr-fr.md [hunk 9]
```diff
@@ -62,12 +100,12 @@ Le ticket d'assistance qui a été créé se présente comme ci-dessous :
...
 > Cher Client,
 >
-> Notre système a détecté que l'adresse **youraddress@domain.com** hébergée sur nos systèmes sous le service **servicename** est source d'envoi de courriers indésirables (spams).
+> Notre système a détecté que l'adresse **youraddress@example.com** hébergée sur nos systèmes sous le service **servicename** est source d'envoi de courriers indésirables (spams).
 > L'envoi d'e-mail a donc été temporairement désactivé.
 >
 > Nous avons actuellement détecté **X** message(s) suspect(s).
 >
-> Afin de nous aider à réactiver l'envoi d'e-mail pour l'adresse : **address@domain.com**,
+> Afin de nous aider à réactiver l'envoi d'e-mail pour l'adresse : **address@example.com**,
```
Comment (Kazarus4) [inline pages/web/emails/locked_for_spam/guide.fr-fr.md:108]:
```suggestion
> Afin de nous aider à réactiver l'envoi d'e-mail pour l'adresse : **address@domain.tld**,
```

File: pages/web/emails/locked_for_spam/guide.fr-fr.md [hunk 10]
```diff
@@ -85,30 +123,13 @@ Dans la continuité de ce message, un échantillon d'en-têtes des e-mails envoy
...
 
-### Étape 3 : répondre aux trois questions posées
-
-> [!warning]
+> [!primary]
 >
-> Le déblocage de l'adresse e-mail ne se fait pas automatiquement. Il est nécessaire de contacter le support, sur le ticket d'assistance, en répondant aux 3 questions posées.
-
-- **Êtes-vous l'émetteur de l'e-mail en question ?** : à l'aide des en-têtes fournis, vérifiez l'expéditeur, le destinataire ainsi que l'objet du message pour vérifier s'il s'agit bien de l'un de vos messages.
+> Une fois que votre ticket a été traité par le support client et que votre adresse e-mail a été débloqué, il est conseillé de modifier le mot de passe de l'adresse e-mail, en veillant à ce qu'il soit suffisamment fort. Vous pouvez utiliser [l'outil de création de mot de passe solide](https://www.cnil.fr/fr/generer-un-mot-de-passe-solide) de la CNIL.
```
Comment (Kazarus4) [inline pages/web/emails/locked_for_spam/guide.fr-fr.md]:
```suggestion
> Une fois que votre ticket a été traité par le support client et que votre adresse e-mail a été débloquée, modifiez le mot de passe de l'adresse e-mail, en veillant à ce qu'il soit suffisamment fort. Vous pouvez utiliser [l'outil de création de mot de passe solide](https://www.cnil.fr/fr/generer-un-mot-de-passe-solide) de la CNIL.
```

File: pages/web/emails/locked_for_spam/guide.fr-fr.md [hunk 11]
```diff
@@ -85,30 +123,13 @@ Dans la continuité de ce message, un échantillon d'en-têtes des e-mails envoy
...
-Si les envois mentionnés dans les en-têtes n'ont pas été initiés par le ou les utilisateurs légitimes de l'adresse e-mail, nous vous invitons à prendre les mesures ci-dessous:
-
-- Effectuer une analyse antivirus de chacun des postes utilisant l'adresse e-mail bloquée pour spam et appliquer un correctif si ces derniers sont infectés.
-
-- Vérifier tous les logiciels utilisant les identifiants de l'adresse e-mail bloquée pour spam (Exemple: télécopieur, logiciel métier, logiciel de messagerie).
-
-- Modifier le mot de passe de l'adresse e-mail, après avoir effectué l'analyse antivirus, en veillant à ce qu'il soit suffisamment fort. Vous pouvez utiliser [l'outil de création de mot de passe solide](https://www.cnil.fr/fr/generer-un-mot-de-passe-solide) de la CNIL.
+## Aller plus loin
 
+Échangez avec notre communauté d'utilisateurs sur <https://community.ovh.com>.
```
Comment (Kazarus4) [inline pages/web/emails/locked_for_spam/guide.fr-fr.md:137]:
```suggestion
Pour des prestations spécialisées (référencement, développement, etc), contactez les [partenaires OVHcloud](https://partner.ovhcloud.com/fr/directory/).

Si vous souhaitez bénéficier d'une assistance à l'usage et à la configuration de vos solutions OVHcloud, nous vous proposons de consulter nos différentes [offres de support](https://www.ovhcloud.com/fr/support-levels/).

Échangez avec notre communauté d'utilisateurs sur <https://community.ovh.com>.
```

File: pages/web/emails/locked_for_spam/guide.fr-fr.md (hunk 1 above)
Comment (OVH-Team-Guides) [inline pages/web/emails/locked_for_spam/guide.fr-fr.md:25]:
```suggestion
Avant de poursuivre et si le blocage concerne une adresse e-mail de type MXplan, identifiez la version que vous possédez pour suivre le bon processus de déblocage. Vérifiez à l'aide du tableau ci-dessous comment distinguer les deux versions.
```

File: pages/web/emails/locked_for_spam/guide.fr-fr.md (hunk 4 above)
Comment (OVH-Team-Guides) [inline pages/web/emails/locked_for_spam/guide.fr-fr.md:35]:
```suggestion
Tout d'abord, assurez-vous, auprès du (des) utilisateur(s) de l'adresse e-mail bloquée, qu'il(s) n'est (ne sont) pas directement lié(s) à l'origine du blocage, suite à une utilisation inhabituelle de l'adresse e-mail (par exemple, la réalisation d'envois massifs d'e-mails). Si c'est le cas, vous devez corriger la situation avant de débloquer l'adresse.
```

File: pages/web/emails/locked_for_spam/guide.fr-fr.md (hunk 5 above)
Comment (OVH-Team-Guides) [reply pages/web/emails/locked_for_spam/guide.fr-fr.md]:
```suggestion
Si l'activité suspecte détectée par l'anti-spam n'a pas été initiée par le(s) utilisateur(s) légitime(s) de l'adresse e-mail, prennez les mesures nécessaires décrites ci-dessous :
```

File: pages/web/emails/locked_for_spam/guide.fr-fr.md [hunk 12]
```diff
@@ -4,54 +4,92 @@ slug: bloque-pour-spam
...
-
-Dirigez-vous vers l'onglet `Comptes e-mail`{.action} de votre plateforme. Si la colonne « statut » à droite de l'adresse e-mail concernée mentionne « Spam », cliquez sur cette mention puis sur `Répondre au ticket`{.action}. Passez ensuite à [l'étape 2](./#etape-2-acceder-au-ticket-dassistance_1){.external} du guide.
-
-![spam](images/blocked-for-SPAM-01-03.png){.thumbnail}
+Si l'activité suspect détecté par l'anti-spam n'a pas été initiée par le ou les utilisateurs légitimes de l'adresse e-mail, nous vous invitons à prendre les mesures ci-dessous:
 
+- Effectuer une analyse antivirus de chacun des postes utilisant l'adresse e-mail bloquée pour spam et appliquer un correctif si ces derniers sont infectés.
 
-### Étape 2 : accéder au ticket d'assistance
+- Vérifier tous les logiciels utilisant les identifiants de l'adresse e-mail bloquée pour spam (Exemple: télécopieur, logiciel métier, logiciel de messagerie).
```
Comment (OVH-Team-Guides) [inline pages/web/emails/locked_for_spam/guide.fr-fr.md]:
```suggestion
- Vérifiez tous les logiciels utilisant les identifiants de l'adresse e-mail bloquée pour SPAM (par exemple : télécopieur, logiciel métier, logiciel de messagerie).
```

File: pages/web/emails/locked_for_spam/guide.fr-fr.md [hunk 13]
```diff
@@ -4,54 +4,92 @@ slug: bloque-pour-spam
...
-#### Pour une adresse e-mail MX plan
-
-Connectez-vous à votre [espace client OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.com/fr/&ovhSubsidiary=fr) et dirigez-vous dans la section `Webcloud`{.action}. Cliquez sur `Emails`{.action}, puis sélectionnez le nom de domaine concerné.
-
-Dirigez-vous vers l'onglet `Comptes e-mail`{.action} de votre plateforme. Si la colonne « statut » à droite de l'adresse e-mail concernée mentionne « Spam », cliquez sur cette mention puis sur `Répondre au ticket`{.action}. Passez ensuite à [l'étape 2](./#etape-2-acceder-au-ticket-dassistance_1){.external} du guide.
-
-![spam](images/blocked-for-SPAM-01-03.png){.thumbnail}
+Si l'activité suspect détecté par l'anti-spam n'a pas été initiée par le ou les utilisateurs légitimes de l'adresse e-mail, nous vous invitons à prendre les mesures ci-dessous:
 
+- Effectuer une analyse antivirus de chacun des postes utilisant l'adresse e-mail bloquée pour spam et appliquer un correctif si ces derniers sont infectés.
```
Comment (OVH-Team-Guides) [inline pages/web/emails/locked_for_spam/guide.fr-fr.md]:
```suggestion
- Effectuez une analyse antivirus de chacun des postes utilisant l'adresse e-mail bloquée pour SPAM et appliquez un correctif si ces derniers sont infectés.
```

File: pages/web/emails/locked_for_spam/guide.fr-fr.md (hunk 6 above)
Comment (OVH-Team-Guides) [reply pages/web/emails/locked_for_spam/guide.fr-fr.md]:
```suggestion
Sélectionnez l'offre e-mail concernée dans les onglets suivants :
```

File: pages/web/emails/locked_for_spam/guide.fr-fr.md (hunk 7 above)
Comment (OVH-Team-Guides) [reply pages/web/emails/locked_for_spam/guide.fr-fr.md]: Not sure we want to add the link here as it's advised not to follow it in this particular case.

File: pages/web/emails/locked_for_spam/guide.fr-fr.md [hunk 14]
```diff
@@ -4,54 +4,92 @@ slug: bloque-pour-spam
...
+|---|---|
+|![email](images/mxplan-starter-legacy-step1.png){.thumbnail}<br> Repérez l'offre dans le cadre « Abonnement »|![email](images/mxplan-starter-new-step1.png){.thumbnail}<br>Repérez la « Référence serveur » dans le cadre « Résumé »|
+
+### Étape 1 : pourquoi votre adresse e-mail est bloquée pour SPAM ? <a name="step1"></a>
+
+Lorsqu'une activité suspecte est détectée au niveau de l'envoi des e-mails, l'adresse concernée est automatiquement bloquée. Dans cette situation vous ne pouvez plus envoyer d'e-mails depuis cette adresse e-mail.
+
+Tout d'abord, assurez-vous, auprès du (des) utilisateur(s) de l'adresse e-mail bloquée, qu'il(s) n'est (ne sont) pas directement lié(s) à l'origine du blocage. Ceci suite à une utilisation inhabituelle de l'adresse e-mail (exemple: réalisation d'envois massifs d'e-mails). Si c'est le cas, vous devrez corriger la situation avant de débloquer l'adresse.
+
+Si l'activité suspecte détectée par l'anti-spam n'a pas été initiée par le (les) utilisateur(s) légitime(s) de l'adresse e-mail, prennez les mesures nécessaires ci-dessous:
```
Comment (OVH-Team-Guides) [inline pages/web/emails/locked_for_spam/guide.fr-fr.md]:
```suggestion
Si l'activité suspecte détectée par l'anti-spam n'a pas été initiée par le (les) utilisateur(s) légitime(s) de l'adresse e-mail, prenez les mesures nécessaires et détaillées ci-dessous :
```

File: pages/web/emails/locked_for_spam/guide.fr-fr.md [hunk 15]
```diff
@@ -4,54 +4,92 @@ slug: bloque-pour-spam
...
-
-![spam](images/blocked-for-SPAM-01-03.png){.thumbnail}
-
-
-### Étape 2 : accéder au ticket d'assistance
-
-Suite à l'étape 1, vous serez alors redirigé vers la fenêtre « Mes demandes d'assistance ». Cliquez sur `...`{.action} à droite du ticket mentionnant l'objet « Account locked for spam.», puis cliquez sur `Voir le détail`{.action}. 
+## En pratique <a name="instructions"></a>
+
+Avant de poursuivre et si le blocage concerne une adresse e-mail de type MXplan, Identifiez la version que vous possédez pour suivre le bon processus de déblocage. Vérifiez à l'aide du tableau ci-dessous comment distinguer les deux versions.
```
Comment (OVH-Team-Guides) [inline pages/web/emails/locked_for_spam/guide.fr-fr.md]:
```suggestion
Avant de poursuivre et si le blocage concerne une adresse e-mail de type MXplan, identifiez la version que vous possédez pour suivre le bon processus de déblocage. Vérifiez, à l'aide du tableau ci-dessous, comment distinguer les deux versions.
```

File: pages/web/emails/locked_for_spam/guide.fr-fr.md [hunk 16]
```diff
@@ -4,54 +4,92 @@ slug: bloque-pour-spam
...
-updated: 2020-04-20
+updated: 2023-06-06
 ---
 
 **Dernière mise à jour le 16/03/2020**
 
 ## Objectif
 
-Si vous avez reçu un e-mail indiquant que l'une de vos adresses e-mail est bloquée pour spam, voici les vérifications et actions à mener.
+Lorsque votre adresse e-mail est bloquée pour SPAM, cela signifie qu'une activité suspecte a été détectée au niveau de l'envoi des e-mails depuis cette adresse. Dans cette situation vous ne pouvez plus envoyer d'e-mail depuis cette adresse e-mail. Vous devrez comprendre pourquoi une activité suspecte a été détectée et réaliser des actions afin d'éviter que cela se reproduise.
```
Comment (OVH-Team-Guides) [inline pages/web/emails/locked_for_spam/guide.fr-fr.md]:
```suggestion
Lorsque votre adresse e-mail est bloquée pour SPAM, cela signifie qu'une activité suspecte a été détectée au niveau de l'envoi des e-mails depuis cette adresse. Dans cette situation, vous ne pouvez plus envoyer d'e-mail depuis cette adresse e-mail. Vous devez alors comprendre pourquoi une activité suspecte a été détectée et réaliser des actions afin d'éviter que cette situation ne se reproduise.
```

File: pages/web/emails/locked_for_spam/guide.fr-fr.md [hunk 17]
```diff
@@ -4,54 +4,92 @@ slug: bloque-pour-spam
...
 
 **Découvrez comment réagir lorsque votre adresse est bloquée pour spam.**
```
Comment (OVH-Team-Guides) [inline pages/web/emails/locked_for_spam/guide.fr-fr.md]:
```suggestion
**Découvrez comment réagir lorsque votre adresse est bloquée pour SPAM.**
```

File: pages/web/emails/locked_for_spam/guide.fr-fr.md [hunk 18]
```diff
@@ -4,54 +4,92 @@ slug: bloque-pour-spam
...
+
+Avant de poursuivre et si le blocage concerne une adresse e-mail de type MXplan, Identifiez la version que vous possédez pour suivre le bon processus de déblocage. Vérifiez à l'aide du tableau ci-dessous comment distinguer les deux versions.
+
+|Version historique de l'offre MX Plan|Nouvelle version de l'offre MX Plan|
+|---|---|
+|![email](images/mxplan-starter-legacy-step1.png){.thumbnail}<br> Repérez l'offre dans le cadre « Abonnement »|![email](images/mxplan-starter-new-step1.png){.thumbnail}<br>Repérez la « Référence serveur » dans le cadre « Résumé »|
+
+### Étape 1 : pourquoi votre adresse e-mail est bloquée pour SPAM ? <a name="step1"></a>
+
+Lorsqu'une activité suspecte est détectée au niveau de l'envoi des e-mails, l'adresse concernée est automatiquement bloquée. Dans cette situation vous ne pouvez plus envoyer d'e-mails depuis cette adresse e-mail.
```
Comment (OVH-Team-Guides) [inline pages/web/emails/locked_for_spam/guide.fr-fr.md]:
```suggestion
Lorsqu'une activité suspecte est détectée au niveau de l'envoi des e-mails, l'adresse concernée est automatiquement bloquée. Dans cette situation, vous ne pouvez plus envoyer d'e-mails depuis cette adresse e-mail.
```

File: pages/web/emails/locked_for_spam/guide.fr-fr.md [hunk 19]
```diff
@@ -4,54 +4,92 @@ slug: bloque-pour-spam
...
+
+Tout d'abord, assurez-vous, auprès du (des) utilisateur(s) de l'adresse e-mail bloquée, qu'il(s) n'est (ne sont) pas directement lié(s) à l'origine du blocage. Ceci suite à une utilisation inhabituelle de l'adresse e-mail (exemple: réalisation d'envois massifs d'e-mails). Si c'est le cas, vous devrez corriger la situation avant de débloquer l'adresse.
```
Comment (OVH-Team-Guides) [inline pages/web/emails/locked_for_spam/guide.fr-fr.md]:
```suggestion
Tout d'abord, assurez-vous, auprès du (des) utilisateur(s) de l'adresse e-mail bloquée, qu'il(s) n'est (ne sont) pas directement à l'origine du blocage, suite à une utilisation inhabituelle de l'adresse e-mail (par exemple, suite à la réalisation d'envois massifs d'e-mails). Si c'est le cas, vous devez corriger la situation avant de débloquer l'adresse.
```

File: pages/web/emails/locked_for_spam/guide.fr-fr.md (hunk 14 above)
Comment (OVH-Team-Guides) [inline pages/web/emails/locked_for_spam/guide.fr-fr.md]:
```suggestion
Si l'activité suspecte détectée par l'anti-spam n'a pas été initiée par le(s) utilisateur(s) légitime(s) de l'adresse e-mail, prenez les mesures nécessaires et détaillées ci-dessous:
```

File: pages/web/emails/locked_for_spam/guide.fr-fr.md [hunk 20]
```diff
@@ -4,54 +4,92 @@ slug: bloque-pour-spam
...
+- Vérifiez tous les logiciels utilisant les identifiants de l'adresse e-mail bloquée pour SPAM (par exemple : télécopieur, logiciel métier, logiciel de messagerie).
+
+### Étape 2 : vérifier le statut de l'adresse e-mail et accéder au ticket d'assistance associé
+
+Sélectionnez l'offre e-mail concernée dans les onglets suivants:
+
+> [!tabs]
+> **Exchange**
+>>
+>> Connectez-vous à votre [espace client OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.com/fr/&ovhSubsidiary=fr) et dirigez-vous dans la section `Webcloud`{.action}. Cliquez sur `Microsoft`{.action} puis sur `Exchange`{.action} et sélectionnez la plateforme Exchange concernée.
```
Comment (OVH-Team-Guides) [inline pages/web/emails/locked_for_spam/guide.fr-fr.md]:
```suggestion
>> Connectez-vous à votre [espace client OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.com/fr/&ovhSubsidiary=fr) et dirigez-vous dans la section `Web Cloud`{.action}. Cliquez sur `Microsoft`{.action} puis sur `Exchange`{.action} et sélectionnez la plateforme Exchange concernée.
```

File: pages/web/emails/locked_for_spam/guide.fr-fr.md [hunk 21]
```diff
@@ -4,54 +4,92 @@ slug: bloque-pour-spam
...
+>> Connectez-vous à votre [espace client OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.com/fr/&ovhSubsidiary=fr) et dirigez-vous dans la section `Webcloud`{.action}. Cliquez sur `E-mail Pro`{.action} puis sélectionnez la plateforme E-mail Pro concernée.
+>> 
+>> Dirigez-vous vers l'onglet `Comptes e-mail`{.action} de votre plateforme. Si la colonne « statut » à droite de l'adresse e-mail concernée mentionne « Spam », cliquez sur cette mention puis sur `Répondre au ticket`{.action}. Le déblocage de l'adresse e-mail ne se fait pas automatiquement. Il est nécessaire de contacter le support, sur le ticket d'assistance, en répondant aux 3 questions posées. <br>
+>> Passez à [l'étape 3](#step3) du guide.
+>> 
+>> ![spam](images/blocked-for-SPAM-01-02.png){.thumbnail}
+>>
+> **MX plan - Nouvelle version**
+>>
+>> Connectez-vous à votre [espace client OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.com/fr/&ovhSubsidiary=fr) et dirigez-vous dans la section `Webcloud`{.action}. Cliquez sur `Emails`{.action}, puis sélectionnez le nom de domaine concerné.
```
Comment (OVH-Team-Guides) [inline pages/web/emails/locked_for_spam/guide.fr-fr.md]:
```suggestion
>> Connectez-vous à votre [espace client OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.com/fr/&ovhSubsidiary=fr) et dirigez-vous dans la section `Web Cloud`{.action}. Cliquez sur `Emails`{.action}, puis sélectionnez le nom de domaine concerné.
```

File: pages/web/emails/locked_for_spam/guide.fr-fr.md [hunk 22]
```diff
@@ -4,54 +4,92 @@ slug: bloque-pour-spam
...
+>> Dirigez-vous vers l'onglet `Comptes e-mail`{.action} de votre plateforme. Si la colonne « statut » à droite de l'adresse e-mail concernée mentionne « Spam », cliquez sur cette mention puis sur `Répondre au ticket`{.action}. Le déblocage de l'adresse e-mail ne se fait pas automatiquement. Il est nécessaire de contacter le support, sur le ticket d'assistance, en répondant aux 3 questions posées.<br>
+>> Passez à [l'étape 3](#step3) du guide.
+>>
+>> ![spam](images/blocked-for-SPAM-01-03.png){.thumbnail}
+>>
+> **MX plan - Historique**
+>>
+>> Si le blocage concerne une adresse e-mail [MXplan version historique](#instructions), il n'y a pas de ticket d'assistance. Veillez bien à consulter [l'étape 1](#step1) de ce guide avant de suivre les instructions suivante.
+>>
+>> Connectez-vous à votre [espace client OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.com/fr/&ovhSubsidiary=fr) et dirigez-vous dans la section `Webcloud`{.action}. Cliquez sur `Emails`{.action}, puis sélectionnez le nom de domaine concerné.
```
Comment (OVH-Team-Guides) [inline pages/web/emails/locked_for_spam/guide.fr-fr.md]:
```suggestion
>> Connectez-vous à votre [espace client OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.com/fr/&ovhSubsidiary=fr) et dirigez-vous dans la section `Web Cloud`{.action}. Cliquez sur `Emails`{.action}, puis sélectionnez le nom de domaine concerné.
```

File: pages/web/emails/locked_for_spam/guide.fr-fr.md [hunk 23]
```diff
@@ -4,54 +4,92 @@ slug: bloque-pour-spam
...
+>> ![spam](images/blocked-for-SPAM-01-04.png){.thumbnail}
+>>
+>> > [!warning]
+>> >
+>> > Dans de rares cas, la colonne « Bloqué pour SPAM » peut indiquer « Non » malgré le fait que l'adresse e-mail soit bloquée. Si vous avez fait le nécessaire pour sécuriser l'adresse e-mail, la solution reste la même que ci-dessus.
+
+
+### Étape 3 : accéder au ticket d'assistance <a name="step3"></a>
+
+Suite à l'étape 2, vous serez alors redirigé vers la fenêtre « Mes demandes d'assistance ». Cliquez sur le bouton `...`{.action} à droite du ticket mentionnant l'objet « Account locked for spam.», puis cliquez sur `Voir le détail`{.action}.
```
Comment (OVH-Team-Guides) [inline pages/web/emails/locked_for_spam/guide.fr-fr.md]:
```suggestion
Suite à l'étape 2, vous serez alors redirigé vers la fenêtre « Mes demandes d'assistance ». Cliquez sur le bouton `...`{.action} à droite du ticket mentionnant l'objet « Account locked for spam.» puis cliquez sur `Voir le détail`{.action}.
```

**Pull request merged**